### DISCUSSION TITLE
网页剪裁

### DISCUSSION THREAD

**Discussion by 210000ling** [obsidianzh/forum#102] [Q&A常见问题]
没有网页剪裁,我都要疯了,用markdownload下载的网页显示效果非常不行.能不能像别家笔记一样,做一个插件剪切网页啊.

**Comment by Ticysis** [comment]
我记得WebClipper好像不错，就是图片要手动更换一下位置，要是不在意的话就更好了

**Comment by cyddgh** [comment]
[如何进行网页剪藏（MOC）](https://publish.obsidian.md/chinesehelp/07+%E4%BF%A1%E6%81%AF%E6%BA%90%E4%B8%8E%E8%B4%A1%E7%8C%AE%E8%80%85/%E5%A6%82%E4%BD%95%E8%BF%9B%E8%A1%8C%E7%BD%91%E9%A1%B5%E5%89%AA%E8%97%8F%EF%BC%88MOC%EF%BC%89)
卡片ID: 202107230946
